entities/README.md

```diff
@@ -0,0 +1,24 @@
+# HTML Entities Generator
+
+This directory contains the tools for generating HTML's [named character references](https://html.spec.whatwg.org/#named-character-references) from [`unicode.xml`](https://github.com/w3c/xml-entities/blob/gh-pages/unicode.xml) and a fixed set of legacy entities.
+
+## Prerequisites
+
+Before building, make sure you have the following commands installed on your system.
+
+- `curl`, `perl`, `python`
+
+## Build
+
+Run the `build.sh` script, like this:
+ ```
+ ./build.sh
+ ```
+
+## Output
```

Comment:
I've made an `out` subdirectory (was tempted to use `output`, but then I'd have to change `.gitignore`, and would rather that we used `out` everywhere). Also updated `README.md`.

No subdirectory for inputs, seems like overkill.